libs/SalesforceSDKCore/SalesforceSDKCore/Classes/UserAccount/SFSDKAuthErrorManager.m

```diff
@@ -161,7 +161,7 @@ - (BOOL)processAuthError:(NSError *)error authContext:(SFSDKAuthSession *)authSe
 + (BOOL)errorIsInvalidAuthCredentials:(NSError *)error
 {
     BOOL errorIsInvalidCreds = NO;
-    if (error.domain == kSFOAuthErrorDomain) {
+    if ([error.domain isEqualToString:kSFOAuthErrorDomain]) {
         if (error.code == kSFOAuthErrorInvalidGrant) {
```

Comment:
Same pointer vs object comparison error on this line?

Reply:
That's not a string, it's an enum value.